example/templates/person.ts

```diff
@@ -31,6 +31,11 @@ const schema = {
     },
     'occupation': {
       'type': 'string'
+    },
+    'slider': {
```

Comment:
can't we name this property more fitting to a person, or simply use the age property?

Reply:
I noticed that when I use range-slider for height for instance, tooltip (icon) doesn't move when you change the data. The reason is there is no left padding in the style of the tooltip. I didn't have this problem when I use it as a separate field in a separate layout in the same example. What might be the problem here?

Reply:
Ok, solved this issue